src/components/Resources/index.js

```diff
@@ -30,17 +51,33 @@ function Resources({ getResourcesUrl }) {
         </Grid>
         <Grid item lg={9}>
           <h2>Resources</h2>
-          <Search label="Search resources" />
+          <Search label="Search resources" search={search} />
+          {searchValue && (
+            <p>
+              You have searched for "<strong>{searchValue}</strong>" and gotten{' '}
```

Comment:
Curious if there's a particular reason why we need to render an extra space here with `{'  '}`? It looks like we just need to space it out; if so, strongly prefer using CSS to handle this or [lean in more on Material UI](https://material-ui.com/components/box/)

```jsx
<p>
    You have searched for 
    <Box component="strong" marginLeft=2>
       {searchValue}
    </Box>
    and gotten
    <Box component="strong" marginLeft=2>
        {resources.count}
    </Box> results.
</p>
```

Reply:
Done in 7bf0858. 

There needed to be a space between the word "for" and the search term in quotes. :P 

![image](https://user-images.githubusercontent.com/4512699/93007433-e6027980-f51d-11ea-9d1b-ac1f61449158.png)